tools/packaging/packaging.psm1

```diff
@@ -34,9 +34,15 @@ function Start-PSPackage {
         [Switch] $SkipReleaseChecks
     )
 
+    $Script:Options = Get-PSOptions
+
     # Runtime and Configuration settings required by the package
     ($Runtime, $Configuration) = if ($WindowsRuntime) {
         $WindowsRuntime, "Release"
+    } elseif ($Type -eq "deb-arm") {
+        New-PSOptions -Configuration "Release" -Runtime "Linux-ARM" -WarningAction SilentlyContinue | ForEach-Object { $_.Runtime, $_.Configuration }
+        # CrossGen isn't supported on ARM yet, so just let the check pass
+        $Script:Options.CrossGen = $true
```

Comment:
Let's not modify `$Script:Options`. `$Script:Options` in `packaging.psm1` is pointing to the same object holding by `$script:Options` in `build.psm1` which is supposed to reflect the real configuration of the latest build.

I suggest to add one if block below to set `crossGenCorrect` in case of `deb-arm`, like
```
if ($Type -eq "deb-arm") {
    $crossGenCorrect = $true
} elseif(-not $IncludeSymbols.IsPresent -and $Script:Options.CrossGen) {
   ....
} elseif ...
```

Reply:
will change